client/src/js/core/frame.js

```diff
@@ -16,14 +16,29 @@ Frame.prototype.motor = function(value) {
     // Multiplying by this value should make possible to write directly to PWM
     // Current range is 0 - 127 with first bit as direction
     // Gets 0-100 
-    let k = 1.27;
+    let k = 1.05;
     return Math.round(Math.abs(value * k) | (value & 0x80));
 };
 
-Frame.prototype.motors = function(speed, directions) {   
-    this.motorsArr.forEach ((motor, index) => {
-        this.motorsArr[index] = Math.abs(speed * 1.27) | (directions[index] << 7);
+Frame.prototype.motors = function (speed, directions, directionList) {
+    const stringDirections = directions.toString();
+    const stringDirectionList = [directionList.left.toString(), directionList.right.toString()];
+    const turn = (stringDirectionList.includes(stringDirections));
+
+    this.motorsArr.forEach((motor, index) => {
+        if (turn) {
+            if (!directions[index]) {
```

Comment:
Is this `if (!directions[index]){` statement needed? Does this cover the case where the direction from `motors.direction[direction]` doesn't have 4 element in the array? Seems like something we shouldn't worry about.

Reply:
`if (!directions[index])` is true when 0 is the value of the index _when turning_.
It doesn't fall in there when the _index = 0.8_
Which is the wheel that should have a _speed_ of _0_.